test-plone-5.1.x.cfg

```diff
@@ -16,3 +16,6 @@ eggs +=
 [versions]
 # Downgrade zc.recipe.egg because 2.0.4 seems to be a broken build.
 zc.recipe.egg = 2.0.3
+
+# The newer versions of Products.LDAPUserFolder require Zope >= 4.0b5
+Products.LDAPUserFolder <= 2.27
```

Comment:
How many projects are affected here?

For customer projects, the pinning should go into the `versions.cfg`.
For `ftw.*` projects I'd also put the pinning into the project and not pin it globally. The purpose of this file is for broader overwrites.

It seems that the same version can be used for Plone 4 and 5, so I suggest to put a constraint into the `ftw.*` package you are working on.
Assuming that the `2.x` series of `Products.LDAPUserFolder` is still working with Plone 4 and 5, but the `3.x` version isn't, the constraint should look like this:
```
'Products.LDAPUserFolder < 3a'
```
The meaning of this constraint is that any `2.x` version is allowed, but no `3.x` version, not even versions such as `3alpha1` (be aware that `3a<3` in setuptools version comparison 😉 )

Reply:
I don't know how many package will be affected. I guess we'll find out then...
Thanks for the help with constraining the package. It works like a charm 😉.